Amethyst/Managers/FocusFollowsMouseManager.swift

```diff
@@ -18,17 +19,39 @@ final class FocusFollowsMouseManager {
     weak var delegate: FocusFollowsMouseManagerDelegate?
 
     private let userConfiguration: UserConfiguration
-    private var mouseMovedEventHandler: Any?
+
+    private var subscription: Disposable? // to work around capture of self in closure
 
     init(userConfiguration: UserConfiguration) {
         self.userConfiguration = userConfiguration
-        mouseMovedEventHandler = NSEvent.addGlobalMonitorForEvents(matching: .mouseMoved) { event in
-            self.focusWindowWithMouseMovedEvent(event)
-        }
+
+        subscription = nil // with this done, we can capture self
+
+        // we want to observe changes to the focusFollowsMouse config, because mouse tracking has CPU cost
+        subscription = UserDefaults.standard.rx.observe(Bool.self, ConfigurationKey.focusFollowsMouse.rawValue)
+            .distinctUntilChanged { $0 == $1 }
+            .scan(nil) { [unowned self] existingHandler, followingIsDesired -> Any? in
+                if let handler = existingHandler {
+                    NSEvent.removeMonitor(handler)
+                    return nil
+                } else if followingIsDesired! {
+                    return NSEvent.addGlobalMonitorForEvents(matching: .mouseMoved) { [unowned self] event in
+                        self.focusWindowWithMouseMovedEvent(event)
+                    }
+                } else {
+                    return nil
+                }
+            }
+            .subscribe()
```

Comment:
Instead of storing the `Disposable` you should keep a `DisposeBag` and use `disposed(by:)`

Reply:
I had tried that initially, but ran into a problem with the closure capturing `self`, hence https://github.com/ianyh/Amethyst/pull/665/files#diff-04ea7b5975998885cc0693ef86642bd3R23

If you see an obvious way around that you'll have to be explicit about it :(

Reply:
I pushed this change, not sure why it worked out better this time :)